torch/cuda/graphs.py

```diff
@@ -68,13 +68,13 @@ def capture_begin(self, pool=None):
             pool (optional): Token (returned by :func:`~torch.cuda.graph_pool_handle` or
                 :meth:`other_Graph_instance.pool()<torch.cuda.CUDAGraph.pool>`) that hints this graph may share memory
                 with the indicated pool.  See :ref:`Graph memory management<graph-memory-management>`.
+            capture_error_mode (str, optional): specifies the cudaStreamCaptureMode for the graph capture stream.
+                Can be "global", "thread_local" or "relaxed". During cuda graph capture, some actions, such as cudaMalloc,
+                may be unsafe. "global" will error on actions in other threads, "thread_local" will only error for
+                actions in the current thread, and "relaxed" will not error on these actions.
```

Comment:
Not sure if this is 100% accurate for relaxed? But tbh the nvidia doc is not 100% clear either.
I would just link to the nvidia doc from there and say that our flags match these ones.

Same comment for the doc below